pkg/metrics/metrics.go

```diff
@@ -40,12 +40,18 @@ func InitializeMetrics() {
 		utils.ForwardBytesGaugeName,
 		forwardBytesGaugeDescription,
 		utils.Direction)
-	WindowsGauge = exporter.CreatePrometheusGaugeVecForMetric(
+	HnsStatsGauge = exporter.CreatePrometheusGaugeVecForMetric(
 		exporter.DefaultRegistry,
 		hnsStats,
 		hnsStatsDescription,
 		utils.Direction,
 	)
+	VfpStatsGauge = exporter.CreatePrometheusCounterVecForMetric(
```

Comment:
Similarly, this should be `VFPStatusGauge`

https://go.dev/wiki/CodeReviewComments#initialisms

Reply:
Thank you @timraymond! Updated the PR